javascript-practice/src/javascripts/services/product.js

```diff
@@ -0,0 +1,18 @@
+import { API_PRODUCTS } from "../constants/url";
+
+export default class APIProduct {
+	constructor() {}
+	/**
+	 * Get all the products in the database from the API.
+	 * @returns {Object} An object containing the HTTP status and product data.
+	 */
+	async get() {
+		const response = await fetch(API_PRODUCTS);
+		const result = await response.json();
```

Comment:
Why we need await here?

Reply:
@mthanh2209 I do not see any changes here. Any comfusion?

Reply:
*confusion

Reply:
1. Because the response.json() method returns a Promise.
2. The response.json() method is an async used to convert JSON data from an HTTP response into a JavaScript object. Because it's an async, it returns a Promise to allow us to handle it in JavaScript's async programming model.
3. If you don't use await, we won't be certain that the JSON data has been successfully converted into a JavaScript object.

Reply:
Clear that